windows/installer.nsi

```diff
@@ -370,6 +402,11 @@ Function FinishPage.Show ; set the user choices if they were remembered
 
     ShowWindow $mui.FinishPage.Run 1
 
+    !if ${BUILD_OPTION} != "jackonwindows"
+        ;Add ASIO label to dialog when not using JACK
+        Call WelcomeFinishShowASIO
+    !endif
```

Comment:
Does it skip the default page? To me it seems a bit strange.

Reply:
The default page is in place, the "call" is just execution the function which add the label to the dialog that contains the ASIO text and only when the build is not for JACK.